Cython/Compiler/Tests/TestCmdLine.py

```diff
@@ -505,7 +520,6 @@ def error(*args):
                 sys.stderr = old_stderr
             self.assertTrue(stderr.getvalue())
 
-        error('-1')
```

Comment:
Why was this removed?

Reply:
Ah - sorry - that was a holdover as I worked out what was being tested.   Here I was confused, because the`-1` flag should generate an error, but there should also be an error for the missing input file name.

Will have a go at the regex check to clarify.

Reply:
I added back the check, with a regex.